parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -41,7 +41,11 @@ installEtcd() {
             docker run --rm --entrypoint cat ${CONTAINER_IMAGE} /usr/local/bin/etcd > "$path/etcd"
             docker run --rm --entrypoint cat ${CONTAINER_IMAGE} /usr/local/bin/etcdctl > "$path/etcdctl"
         else
-            img unpack -o "$path" ${CONTAINER_IMAGE}
+            # img unpack requires a non-existent dirctory
+            tmpdir=/root/etcd${RANDOM}
+            img unpack -o ${tmpdir} ${CONTAINER_IMAGE}
+            mv ${tmpdir}/usr/local/bin/etcd ${tmpdir}/usr/local/bin/etcdctl /usr/bin
```

Comment:
let's re-use the existing `$path` var instead of referring to `/usr/bin` explicitly, even though it's somewhat academic (i.e., this stuff hasn't been validated on coreos at all)

Reply:
Done!